src/components/TodoItem.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable react/prop-types */
+import React, { useState, useEffect } from 'react';
+import { FaTrash } from 'react-icons/fa';
+import styles from './TodoItem.module.css';
+
+const TodoItem = ({
+  todo, handleChangeProps, deleteTodoProps, setUpdate,
+}) => {
+  const [editing, setEditing] = useState(false);
+
+  useEffect(() => () => {
+  }, []);
```

Comment:
[OPTIONAL] Kindly remove unused function, because it would make your project clearner.